### PR TITLE
Tailwind: dialog component

### PR DESCRIPTION
**Pull request by sarahgm**

### CODE REVIEW

File: packages/components/src/Dialog/Dialog.stories.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /* eslint-disable react-hooks/rules-of-hooks */
-import React, { useState } from 'react';
+import React from 'react';
 import type { Meta, StoryObj } from '@storybook/react';
+import { useState } from '@storybook/addons';
```
Comment (sebald) [inline packages/components/src/Dialog/Dialog.stories.tsx:4]:
```suggestion
```

File: packages/components/src/Dialog/Dialog.stories.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 /* eslint-disable react-hooks/rules-of-hooks */
-import React, { useState } from 'react';
+import React from 'react';
```
Comment (sebald) [inline packages/components/src/Dialog/Dialog.stories.tsx:2]:
```suggestion
import React, { useState } from 'react';
```

File: packages/components/src/Dialog/Dialog.test.tsx [hunk 3]
```diff
@@ -426,17 +420,19 @@ test('dialog supports size', () => {
...
 
 test('renders with dialog controller', () => {
   render(
     <OverlayProvider>
-      <Dialog.Controller open>
-        <Dialog>
-          <Headline>Headline</Headline>Content
-        </Dialog>
-      </Dialog.Controller>
+      <ThemeProvider theme={theme}>
```
Comment (sebald) [inline packages/components/src/Dialog/Dialog.test.tsx:429]: use test utils `render`!

**Review by sebald — CHANGES_REQUESTED** [review]

File: packages/components/src/Dialog/Dialog.test.tsx (hunk 3 above)
Comment (sarahgm) [reply packages/components/src/Dialog/Dialog.test.tsx:429]: ok ok

File: packages/components/src/Dialog/Dialog.test.tsx [hunk 4]
```diff
@@ -1,58 +1,50 @@
 /* eslint-disable testing-library/no-node-access */
 /* eslint-disable testing-library/no-container */
 import React, { useState } from 'react';
-import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import { fireEvent, screen, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
 import { OverlayProvider } from '@react-aria/overlays';
-import { ThemeProvider } from '@marigold/system';
+import { Theme, ThemeProvider } from '@marigold/system';
```
Comment (sebald) [inline packages/components/src/Dialog/Dialog.test.tsx:8]:
```suggestion
import { Theme } from '@marigold/system';
```

**Review by sebald — APPROVED** [review]

**Pull request merged**